Algorithm/Number_theory/readme.md

```diff
@@ -39,3 +39,4 @@
 # Questions related to Number Theory
 
 - GCD of two number using Euclidean Algorithm ----> [Java](/Code/Java/GCD.java)
+- Counting number of digits in factorial using Kamenetsky's Formula ----> [Java](Code/Java/Kamenetsky_Formula.java)
```

Comment:
add the name in the sorted alphabetical order, i.e. Counting.... should come before GCD...

Reply:
made the change.